docs/discovery.md

```diff
@@ -194,7 +194,7 @@ swarm is connected to the public internet. To create your cluster:
 
     This can be on any machine or even your laptop.
 
-        $ swarm manage -H tcp://<swarm_ip:swarm_port> token://<cluster_id>
+        $ swarm manage -H <swarm_ip:swarm_port> token://<cluster_id>
```

Comment:
@allencloud Yes, it seems `tcp://` doesn't affect the functions. So could u just merge the second commit and ignore the first?